jib-cli/src/test/java/com/google/cloud/tools/jib/cli/jar/JarProcessorTest.java

```diff
@@ -44,4 +51,77 @@ public void testDetermineJarType_standard() throws IOException, URISyntaxExcepti
     JarType jarType = JarProcessor.determineJarType(standardJar);
     assertThat(jarType).isEqualTo(JarType.STANDARD);
   }
+
+  @Test
+  public void testExplodeMode_standard() throws IOException, URISyntaxException {
+    Path standardJar = Paths.get(Resources.getResource(STANDARD_RESOURCE_DIR_WITH_CP).toURI());
+    List<FileEntriesLayer> layers = JarProcessor.explodeStandardJar(standardJar);
+    FileEntriesLayer classesLayer = layers.get(0);
+    FileEntriesLayer resourcesLayer = layers.get(1);
+    FileEntriesLayer dependenciesLayer = layers.get(2);
+
+    assertThat(layers.size()).isEqualTo(3);
```

Comment:
Assert before attempting `layers.get(2)`. Otherwise, for example, if the code base accidentally changes to produce only two layer, we'll hit an index-out-of-bound error when attempting `.get(2)`.

Reply:
That's a good idea!